### DISCUSSION TITLE
About the release for Godot 4

### DISCUSSION THREAD

**Discussion by CharlieMalphas** [Techokami/SonicWorldsNext#38] [Q&A] [answered]
Thank you very much for taking the time to answer the questions I posted on March 22. I would be glad to know if you have a date set for the release of this framework for Godot 4.

**Comment by Renhoex** [comment, marked as answer]
We can't estimate an exact date but the latest I wanna aim for is SAGE 2023 if not earlier. It really depends how fast we can iron out all the new bugs.

**Reply by CharlieMalphas** [reply]
Thank you for your answer. I wish you all the best in achieving your goals!
